reddit_decider/__init__.py

```diff
@@ -252,18 +244,16 @@ def make_object_for_context(self, name: str, span: Span) -> "Decider":
         user_event_fields = ec.user.event_fields()
         try:
             decider_context = DeciderContext(
-                user_id=ec.user.id,
-                loid=ec.loid.id,
+                user_id=user_event_fields.get("user_id"),
+                logged_in=user_event_fields.get("logged_in"),
```

Comment:
getting `user_id` & `logged_in` from `ec.user.event_fields()` instead of off `edge_context.user` directly to match [current functionality](https://github.com/reddit/experiments.py/blob/ca417f1b3d29fa39fa95d483315bf4b7530ff546/reddit_experiments/__init__.py#L249-L252).